docs/hive-integration/platform-selection.md

```diff
@@ -0,0 +1,16 @@
+# Platform Selection
+
+The Assisted Installer currently supports the following OpenShift platforms:
+- BareMetal
+- VSphere
+- None
+
+Select the platform in the AgentClusterInstall CR, via `spec.platformType`.
+
+In some cases, the system will automatically set the platform type:
+- The default platform for multi-node clusters is be BareMetal
+- The default platform for Single Node OpenShift is None
+- Enabling `spec.networking.userManagedNetworking` will cause the platform to be None
+
```

Comment:
Is it worth adding some specific details on using vSphere here? Such as no SNO support for now. And similar to BareMetal, it uses UPI installation in regards to VMs provisioning should be done by the user. Also, VIPs DHCP allocation should be configured manually in vSphere env, maybe just a gritty detail:)

Reply:
This is out of scope of what I wanted to document in this PR.  You are welcome to add it in another PR :)